File: ts-sdk/src/sdk/models/operations/updateautoapplyfordeployrequest.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";


export class UpdateAutoApplyForDeployRequestPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=database" })
  database: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=number" })
  number: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;
}

export class UpdateAutoApplyForDeployRequestRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: UpdateAutoApplyForDeployRequestPathParams;
}

export class UpdateAutoApplyForDeployRequestResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  updateAutoApplyForDeployRequest200ApplicationJSONObject?: Record<string, Record<string, any>>;
}

File: go-sdk/pkg/models/operations/updateautoapplyfordeployrequest.go
package operations

type UpdateAutoApplyForDeployRequestPathParams struct {
	Database     string `pathParam:"style=simple,explode=false,name=database"`
	Number       string `pathParam:"style=simple,explode=false,name=number"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type UpdateAutoApplyForDeployRequestRequest struct {
	PathParams UpdateAutoApplyForDeployRequestPathParams
}

type UpdateAutoApplyForDeployRequestResponse struct {
	ContentType                                             string
	StatusCode                                              int
	UpdateAutoApplyForDeployRequest200ApplicationJSONObject map[string]map[string]interface{}
}
